### DISCUSSION TITLE
Is the project still alive?

### DISCUSSION THREAD

**Discussion by GaroK** [Try/OpenGothic#154] [General]
Are there any plans to do anything more with the project?

**Comment by Try** [comment]
@GaroK commooooooon, what kind of question is it?)
![изображение](https://user-images.githubusercontent.com/122894/121775343-409d3480-cb87-11eb-9835-939065e41cfa.png)

**Comment by Try** [comment]
For next release there 3 main thing to address:
 1. Dynamic physics: mostly working, need to address tunneling and performance issues 
 2. Spell effect - this is a long running issue,  because it's hard to match every single one with original game, take a lot of time to test it
 3. Planning to fix long known bugs with inventory and animation
